components/stash/stash.less

```diff
@@ -2,12 +2,14 @@
   z-index: 4;
   margin-left: 20px;
   margin-right: 20px;
+  margin-bottom: -15px;
   background: #55323C;
   h4 {
     margin-top: 0px;
   }
-  margin-bottom: -15px;
-
+  .toggle-show-commit-diffs {
```

Comment:
is this related to some test failing now?

Reply:
No, the stash tests did fail before on macOS which is fixed by this.
The current test failures (mainly on macOS) are different, but I don't have a mac available so they are pretty hard to debug for me.

Reply:
Just want to note that macOS tests sometimes succeed but yes, they are currently the most flaky.
https://github.com/campersau/ungit/runs/639350614